Achmad Ramadhan Tamherwarin

achrams opened the pull request:
Client : https://fancytodo-client-949cc.web.app/

achrams committed:
Finish

NODE_ENV=development
SECRET=lalalisa
PORT=3000
AP_ID=$159cd39e812f312e3f6a18f3af0b26ed
ACCESS_KEY=13fec12a51ef644590508255798030da
CLIENT_ID=94289824709-9elnoiuecs7tfdilbrmos1gus49umpr6.apps.googleusercontent.com
DEFAULT_PASSWORD=blackpink

achrams committed:
Update README.md

ziadyjo commented:
Ini data env untuk server ya mungkin salah isi comment pull request

ziadyjo commented on main.js:
```diff
@@ -0,0 +1,405 @@
+                    auth()
+                })
+                .fail((err) => {
+                    Swal.fire({
```
tolong di cek lagi, tambah todo di hari sekarang kenapa ga bisa ya

achrams replied:
okee mas udah bisa sekarang, problemnya di server ternyata

ziadyjo commented on main.js:
```diff
@@ -0,0 +1,405 @@
+                    <option value="Done" >Done</option>
+                    </select>
+                    <label>due date</label>
+                    <input id="swal-input4-edit" class="swal2-input" type="date" values="${result.due_date}">`,
```
tanggalnya belum berhasil tampil ya begitu diubah

achrams replied:
iya mas, tanggalnya beda pas di show, itu kenapa ya

achrams replied:
format tanggalnya harus dibikin ke UTC ternyata mas, kalo ngga ntar bulannya kurang sebulan

ziadyjo commented on main.js:
```diff
@@ -0,0 +1,405 @@
+        })
+}
+
+function showSchedule() {
```
fungsi ini langsung dapet data dari 3 api kah atau baru berapa api yang dipake?

achrams replied:
iya mas, 3 api jadi 1 data

achrams committed:
Fixing Errors

achrams committed:
Update README.md

achrams committed:
Deployed

+ client : https://fancytodo-client-949cc.web.app/
+ server: https://achrams-fancy-todo.herokuapp.com/

achrams committed:
deployed

ziadyjo commented:
keren udh deploy ya

ziadyjo commented:
goodjob!
